edenia/eden-spend-explorer
# fix(webapp): add enviroment variable

### Add enviroment variable

### What does this PR do?

- Resolve #351

### Steps to test

1. Run Project 
2. Enter to delegate or try login, must see this: 
![imagen](https://user-images.githubusercontent.com/49013556/209371741-d8169c33-8f46-428e-b82a-e2ef04b5aafb.png)


#### CheckList
